pkg/digger/digger_config.go

```diff
@@ -46,6 +53,40 @@ type Workflow struct {
 	Configuration *WorkflowConfiguration `yaml:"workflow_configuration"`
 }
 
+type DirWalker interface {
+	GetDirs(workingDir string) ([]string, error)
+}
+
+type MockDirWalker struct {
+	Files []string
+}
+
+type FileSystemDirWalker struct {
+}
+
+func (walker *FileSystemDirWalker) GetDirs(workingDir string) ([]string, error) {
+	var files []string
+	err := filepath.Walk(workingDir,
```

Comment:
do we need to walk whole tree? or just first level? If they have subfolders in subfolders they will be treated as different projects. Maybe we should only support one level

Reply:
I think we need to walk the whole tree, otherwise 'include' pattern will be limited only to one dir name (first level) ?